attribution_scopes.md

```diff
@@ -0,0 +1,229 @@
+
+```jsonc
+// trigger registration 1 for product related to products 2 and 3 at t=3
+{
+  ..., // existing fields
+  "attribution_scopes": ["product2", "product3"]
+}
+```
+
+In this example the API will start by finding any sources that have an `attribution_scopes` that matches at least one of the trigger registration `attribution_scopes`. In this example that would be source registration 1 and 2. The API will then perform attribution between these two sources. In this example source registration 2 will win the attribution process because it was the most recent source registration. The API caller would receive an attribution report attributing the trigger registration to source registration 2.
+
+## Alternatives Considered
+
+One alternative that was considered was instead of using `attribution_scopes` the API would perform attribution matching on a finer granularity than site (e.g. origin or path). This approach would allow the rest of the API logic to continue working as it does currently. However, the `attribution_scopes` proposal gives users additional flexibility in case their URL structures do not match their logical scope structures. Additionally, matching on finer granularity than site may open additional potential attack vectors and would require additional rate limits.
+
+## Privacy Considerations
+
+### Impact on flexible event-level reporting
+
+The use of `attribution_scopes` will have some impact on the amount of utility that can be extracted from the [Flexible Event-Level](https://github.com/WICG/attribution-reporting-api/blob/main/flexible_event_config.md) reporting proposal. In order to preserve the current privacy levels, the information gain limit that is used to verify acceptable flexible event-level configurations will now include the `attribution_scope_limit` and `max_event_states` values as part of the calculation. In other words, the additional information gain from the `attribution_scope_limit` and `max_event_states` that are chosen will be subtracted from the flexible event-level information gain limit and then used to identify whether the flexible event-level configuration is within the current privacy levels.
+
+TODO: Update [script](https://github.com/WICG/attribution-reporting-api/tree/main/ts) to allow for testing different configurations that include `attribution_scope_limit`
+
+Note that the `attribution_scope_limit` does not have any impact on the privacy mechanisms used in Aggregatable Reports.
+
+### Additional Privacy Limits
+
+It is possible for an adversary to register multiple navigation sources during source registration, and use these multiple sources, each with a different attribution scopes value, to gain additional information about a user based on which attribution scope is chosen. To prevent this abuse the number of unique attribution scope sets per reporting origin per navigation needs to be limited.
```

Comment:
> To prevent this abuse the number of unique attribution scope sets per reporting origin per navigation needs to be limited.

Could an attacker register multiple origins to avoid the limit?